src/transforms/tree.js

```diff
@@ -3,28 +3,42 @@ import {ascendingDefined} from "../defined.js";
 import {column, identity, isObject, one, valueof} from "../options.js";
 import {basic} from "./basic.js";
 
-export function treeNode({
-  path = identity, // the delimited path
-  delimiter, // how the path is separated
-  frameAnchor,
-  treeLayout = tree,
-  treeSort,
-  treeSeparation,
-  treeAnchor,
-  ...options
-} = {}) {
+/**
+ * Based on the tree options described above, populates the **x** and **y** channels with the positions for each node. The following defaults are also applied: the default **frameAnchor** inherits the **treeAnchor**. This transform is intended to be used with [dot](https://github.com/observablehq/plot/blob/main/README.md#dot), [text](https://github.com/observablehq/plot/blob/main/README.md#text), and other point-based marks. This transform is rarely used directly; see the [Plot.tree compound mark](https://github.com/observablehq/plot/blob/main/README.md#plottreedata-options).
```

Comment:
Here's an example where the reference ("the tree options described above") fails to work when viewed outside of the context of README.md (it doesn't have to be addressed in this PR, of course!).

Reply:
good catch 🙏